src/legacy/ui/public/vis/editors/default/default.js

```diff
@@ -57,7 +58,7 @@ const defaultEditor = function ($rootScope, $compile) {
       }
     }
 
-    render({ uiState, timeRange, filters, appState }) {
+    render({ uiState, timeRange, filters, query }) {
```

Comment:
what's the difference? Relying on `getAppState` instead of using the one that got passed in seems like a step backwards into angular code.

Reply:
we need to get rid of appState all together, so it doesn't make much difference here.